### DISCUSSION TITLE
How do I treat an app that no longer support 32 bit or somehow have different version?

### DISCUSSION THREAD

**Discussion by LazyGeniusMan** [ScoopInstaller/Scoop#4618] [Help and Questions] [answered]
For example, an App release a new version 1.1, but it's no longer support 32 bit, so the 32 bit latest version is 1.0 and 64 bit latest version is 1.1

- Does sccop support `version.architecture` in app manifest?
- Should I freeze the 32 bit version in `autoupdate.architecture.32bit.url` by hardcode the version instead of using `$version` ?

**Comment by rashil2000** [comment, marked as answer]
> For example, an App release a new version 1.1, but it's no longer support 32 bit, so the 32 bit latest version is 1.0 and 64 bit latest version is 1.1

For such cases, we drop 32bit downloads. If the author of the application has skipped 32bit versions for a new release, it's unlikely that they'll add support for 32bit PCs in subsequent releases. 

> * Does sccop support `version.architecture` in app manifest?

No. 

> * Should I freeze the 32 bit version in `autoupdate.architecture.32bit.url` by hardcode the version instead of using `$version` ?

Yes, you can do that. But add a note in the manifest that 32bit version is different from 64bit.
